modules/assim.batch/R/pda.utils.R

```diff
@@ -688,18 +726,17 @@ pda.postprocess <- function(settings, con, params, pname, prior, prior.ind, burn
 
 
   ## coerce parameter output into the same format as trait.mcmc
-  # ********* TODO: Check/fix... do we really want to save mcmc for only the parameters that were updated???
   pname <- rownames(post.distns)
   trait.mcmc <- list()
-  for(i in prior.ind){
-    beta.o <- array(params[,i],c(nrow(params),1))
+  for(i in 1:length(prior.ind)){
```

Comment:
Note this will produce unexpected (bad) results if `prior.ind` doesn't have length at least 1; can this be guaranteed? Some defensive programming might be good: use assertions or `stopifnot` to check beforehand.